stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/channel/ChannelController.kt

```diff
@@ -305,76 +304,24 @@ public class ChannelController internal constructor(
         runChannelQuery(QueryChannelPaginationRequest(limit).toWatchChannelRequest(domainImpl.userPresence))
     }
 
-    private fun getLoadMoreBaseMessageId(direction: Pagination): String? {
-        val messages = mutableState.sortedMessages.value
-        return if (messages.isNotEmpty()) {
-            when (direction) {
-                Pagination.GREATER_THAN_OR_EQUAL,
-                Pagination.GREATER_THAN,
-                -> {
-                    messages.last().id
-                }
-                Pagination.LESS_THAN,
-                Pagination.LESS_THAN_OR_EQUAL,
-                -> {
-                    messages.first().id
-                }
-            }
-        } else {
-            null
-        }
-    }
-
-    /**
-     *  Loads a list of messages before the oldest message in the current list.
-     */
+    /** Loads a list of messages before the oldest message in the current list. */
     internal suspend fun loadOlderMessages(limit: Int = 30): Result<Channel> {
-        return runChannelQuery(
-            QueryChannelPaginationRequest(limit).apply {
-                getLoadMoreBaseMessageId(Pagination.LESS_THAN)?.let {
-                    messageFilterDirection = Pagination.LESS_THAN
-                    messageFilterValue = it
-                }
-            }.toWatchChannelRequest(domainImpl.userPresence)
-        )
+        return runChannelQuery(channelLogic.olderWatchChannelRequest(limit, null))
```

Comment:
Maybe we could use named parameters for these functions, just so we know what the null param is specifically

If you feel like it's not important, than that's also okay! :]